### PR TITLE
Matplotlib 2.0 deprecation

### PR DESCRIPTION
**Pull request by arthurmensch**
I just adressed the annoying messages from matplotlib 2.0. Works with matplotlib down to 1.3

### CODE REVIEW

File: nilearn/plotting/displays.py [hunk 1]
```diff
@@ -721,7 +720,11 @@ def _show_colorbar(self, cmap, norm, threshold=None):
                          x_adjusted_width,
                          height - (self._colorbar_margin['top'] +
                                    self._colorbar_margin['bottom'])]
-        self._colorbar_ax = figure.add_axes(lt_wid_top_ht, axis_bgcolor='w')
+        self._colorbar_ax = figure.add_axes(lt_wid_top_ht)
+        if matplotlib.__version__ >= LooseVersion("1.4"):
```
Comment (GaelVaroquaux) [inline nilearn/plotting/displays.py]: I think that the right way to do the comparison is LooseVersion(matplotlib.__version__) >= LooseVersion("1.4")

(same comment below)

**Comment by GaelVaroquaux** [comment]
Looks good to merge once the LooseVersion thing is addressed.

**Comment by GaelVaroquaux** [comment]
You did a typo in the current code: "__version__" not "version". The tests are all failing :)

File: nilearn/plotting/displays.py (hunk 1 above)
Comment (arthurmensch) [inline nilearn/plotting/displays.py]: There is no version field in matplotlib lib

**Comment by GaelVaroquaux** [comment]
The code seems broken under MPL 1.4. Check out travis.

**Comment by GaelVaroquaux** [comment]
LGTM. Merging. Thanks a lot!

**Pull request merged**